### PR TITLE
New formula: PL/R 8.3.0.16.

### PR DESCRIPTION
**Pull request by lifepillar**
### All Submissions:

- [X] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew/blob/master/.github/CONTRIBUTING.md) document?
- [X] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew/pulls) for the same update/change?

### New Formulae Submissions:

- [X] Does your submission pass
`brew audit --strict --online <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [X] Have you built your formula locally prior to submission with `brew install <formula>`?

### CODE REVIEW

File: Library/Formula/plr.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
+class Plr < Formula
+  desc "PL/R - R Procedural Language for PostgreSQL"
+  homepage "http://www.joeconway.com/plr/"
+  url "https://github.com/jconway/plr/archive/REL8_3_0_16.tar.gz"
+  sha256 "57e2384f7b51328c9e6d92a40039cae7ac3e187ece03a1d33985b751f24bfe18"
+  head "https://github.com/jconway/plr.git"
+
+  depends_on "pkg-config" => :build
+  depends_on "postgresql"
+  depends_on "r"
```
Comment (DomT4) [inline Library/Formula/plr.rb]: `r` lives in [Homebrew/science](https://github.com/Homebrew/homebrew-science/blob/master/r.rb) so if this needs that this formula needs to live there also.

**Comment by lifepillar** [comment]
Ops, sorry.

File: Library/Formula/plr.rb (hunk 1 above)
Comment (apjanke) [inline Library/Formula/plr.rb]: Yeah. This doesn't just need `r`, it's an extension/embedded version of R. This should go in `homebrew/science`.

**Comment by apjanke** [comment]
No worries! That might be our fault: It looks like we don't have the R => `homebrew/science` or X11 => `homebrew/x11` requirements documented in the [Acceptable Formulae guidelines page](https://github.com/Homebrew/homebrew/blob/1cff6238a3ad5290170558836cdcf734bbcf0139/share/doc/homebrew/Acceptable-Formulae.md), and we probably should, since this otherwise passes the guidelines.

You can just resubmit the exact same formula over there and it should go through no problem.

**Pull request closed without merging**